src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs

```diff
@@ -34,10 +35,17 @@ public ActivitiesController(IMapper mapper, IMediator mediator)
         [Route("latest")]
         public ActionResult Latest(GetLatestActivitiesQuery query)
         {
-            var response = _mediator.Send(query);
-            var model = _mapper.Map<LatestActivitiesViewModel>(response);
+            try
```

Comment:
Is it worth adding a unit test for the try catch logic?

Reply:
Done